.github/styles/proselint/RealWorld.yml

```diff
@@ -0,0 +1,6 @@
+extends: existence
+message: "Consider removing '%s'."
+ignorecase: true
+level: warning
+tokens:
+  - 'real world'
```

Comment:
We don't need to say `real world examples`

Reply:
lol

Reply:
Saving the review for my future self

Reply:
![giphy](https://user-images.githubusercontent.com/26760571/207679553-e48e2919-bda0-483b-82f7-b6d2f8984771.gif)